### ISSUE TITLE
JsonTypeInfo.As.PROPERTY does not work

### ISSUE THREAD

**Issue by jigarparekh80** [stoicflame/enunciate#388]
if i specify 

```java
@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "class")
```

then example does not show property class. this is working correctly for WRAPPER_OBJECT and WRAPPER_ARRAY.

**Comment by stoicflame** [comment]
Fixed at 875c579.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
